Source/Core/Core/HW/DVDInterface.cpp

```diff
@@ -1291,7 +1343,7 @@ u64 SimulateDiscReadTime(u64 offset, u32 length)
 		}
 	}
 
-	g_last_read_offset = (offset + length - 2048) & ~2047;
+	g_last_read_offset = end_offset;
```

Comment:
What's the point of the "- 2048"? Isn't g_last_read_offset supposed to be the last offset that has been read, not an offset before it?